src/components/forms/material-form/material-form.js

```diff
@@ -164,18 +152,22 @@ function MaterialForm({ material, id }) {
     }
   ];
 
-  const languageTabs = languages.map((lang) => (
-    <Tab
-      className={
-        (touched[`${lang}Description`] && errors[`${lang}Description`]) ||
-        (touched[`${lang}Name`] && errors[`${lang}Name`])
-          ? styles.errorTab
-          : styles.tabs
-      }
-      label={lang}
-      key={lang}
-    />
-  ));
+  const languageTabs = languages.map((lang) => {
+    const touchedAndErrorsDescription =
+      touched[`${lang}Description`] && errors[`${lang}Description`];
+    const touchedAndErrorsName =
+      touched[`${lang}Name`] && errors[`${lang}Name`];
+    const tabConditionForStyles =
+      touchedAndErrorsDescription || touchedAndErrorsName;
```

Comment:
It is the same about using an "if statement" and one variable

Reply:
I don't have ternaries here anymore. It's already removed to utils file. I only define and use these additional variables because they are needed for local map() calculations